resources/language/messages.pot

```diff
@@ -2252,3 +2252,7 @@ msgstr ""
 msgctxt "#30702"
 msgid "Exceeded daily allowed subtitles downloads"
 msgstr ""
+
+msgctxt "#30703"
+msgid "Can be used by Burst (see its settings) or for debugging"
```

Comment:
I think it should not mention Burst, as it can be used by any search addon, not only Burst.

Reply:
fixed

Reply:
@elgatito ok, i change "burst" to "search providers".